src/kernel/outwards.jl

```diff
@@ -27,15 +27,14 @@ function OutwardsDispersal{R,W}(; neighborhood=DispersalKernel{3}()) where {R,W}
     OutwardsDispersal{R,W}(neighborhood)
 end
 
-@inline function applyrule!(data, rule::OutwardsDispersal{R,W}, state, I) where {R,W}
-    state == zero(state) && return
-    sum = zero(state)
+@inline function applyrule!(data, rule::OutwardsDispersal{R,W}, N, I) where {R,W}
+    N == zero(N) && return
```

Comment:
`return nothing` to be BlueStyle

Reply:
Yes exactly!! Thanks for picking that up. I think `return nothing` is clearer too.